src/guacamole-docker.md

```diff
@@ -646,6 +646,41 @@ valid Docker variables for enabling and configuring header authentication:
   header that will be used used to authenticate the user to Guacamole. If this
   is not specified the default value of REMOTE_USER will be used.
 
+(guacamole-docker-tomcat-remote-ip-valve)=
+
+### Execution behind a proxy
+
+To run Guacamole behind a reverse proxy, the Tomcat's [`RemoteIpValve`](https://tomcat.apache.org/tomcat-8.5-doc/config/valve.html#Remote_IP_Valve) as in [](tomcat-remote-ip).
```

Comment:
I'm not entirely sure what you're trying to say, here, so I think this could use some clarification. You don't _need_ to set the `RemoteIpValve` in order for Guacamole to work correctly behind a reverse proxy. You need to set it if Guacamole is running behind a reverse proxy and you wish to have the client IP address accurately recorded by Guacamole (subject to several caveats mentioned in the proxy section).